src/lib/api/requests/index.ts

```diff
@@ -1,2 +1,2 @@
-export { getPhotos } from './get-photos'
+export { getPhotos } from './get-photo'
```

Comment:
Тут вроде что-то не то с импортами, вероятно так
```suggestion
export { getPhoto } from './get-photo'
```

Reply:
Уже убрал